app/components/embed/media_with_companion_windows_component.html.erb

```diff
@@ -90,7 +90,16 @@
           <div class="header-background">
             <h3>Transcript</h3>
           </div>
-          <div class="transcript" data-action="media-loaded@window->transcript#persistPlayer">
+          <div class="autoscroll">
+            <div class="form-check float-left">
+              <label class="form-check-label" for="autoscroll">Autoscroll</label>
+              <input data-transcript-target="autoscroll" data-action="transcript#toggleAutoscroll" type="checkbox" class="form-check-input" id="autoscroll" checked>
+            </div>
+            <div class="float-right">
```

Comment:
In my branch, I was struggling to find these CSS classes defined (form-*, float-*). They look like bootstrap classes but they looked undefined when I loaded the page. Maybe we need to add an `@import` in media.scss?

Reply:
Sounds good!